amps/ags/rm-community/rm-community-repo/source/java/org/alfresco/repo/action/ExtendedActionServiceImpl.java

```diff
@@ -49,12 +53,17 @@
  */
 public class ExtendedActionServiceImpl extends ActionServiceImpl implements ApplicationContextAware
 {
+    private static final Set<String> ALLOWED_CONTEXTS_FOR_RM_ACTIONS = Set.of("rules");
```

Comment:
Any ideas of how to do it better? I feel kinda bad with comparing values through a string variable and not at least a static variable if not even an enum, but I also don't want to add any unneeded dependencies. I don't know how the repo is structured, is there a place to add such an enum/static?